### DISCUSSION TITLE
About Advanced Use-Cases in Doc

### DISCUSSION THREAD

**Discussion by yinlokc** [mProjectsCode/obsidian-meta-bind-plugin#223] [Q&A]
I found the following code from https://www.moritzjung.dev/obsidian-meta-bind-plugin-docs/guides/advancedusecases/ and the result is INPUT[number:list[0]], I should this will be render as "1"? My question is how to render (in the code) and output 1 based on the code below?

---
list:
  - 1
  - 2
  - 3
index: 0
---

Index: `INPUT[inlineSelect(option(0), option(1), option(2)):index]`

```meta-bind-js-view
{index} as index
---
const str = `\`INPUT[number:list[${context.bound.index}]]\``;
return engine.markdown.create(str)
```

**Comment by mProjectsCode** [comment]
This should behave correctly in `0.12.4`
